src/fileTransformers/prismaSchemaProvider.ts

```diff
@@ -1,12 +1,16 @@
 import { FileTransformer } from '../fileTransformer/fileTransformer'
 
 export const prismaSchemaProvider: FileTransformer = (params) => {
-  const { file, parameters } = params
+  const { file, parameters, tools } = params
 
   let content = file.content
 
   if (file.path === 'prisma/schema.prisma') {
-    content = content.replace(`provider = "postgresql"`, `provider = "${parameters.datasourceProvider}"`)
+    content = tools.replaceContent({
```

Comment:
Using this new tiny api in a few places to make our transformers a bit safer. Low effort for some benefit. We'll get better feedback when  snapshot tests fail as well as when adding new snapshots.